packages/roosterjs-editor-core/lib/coreApi/getSelectionRangeEx.ts

```diff
@@ -0,0 +1,83 @@
+import {
+    EditorCore,
+    GetSelectionRangeEx,
+    SelectionRangeEx,
+    TableSelectionPluginState,
+} from 'roosterjs-editor-types';
+import {
+    contains,
+    createRange,
+    findClosestElementAncestor,
+    NormalSelectionRange,
+    safeInstanceOf,
+    TableSelectionRange,
+} from 'roosterjs-editor-dom';
+
+/**
+ * @internal
+ * Get current or cached selection range
+ * @param core The EditorCore object
+ * @param tryGetFromCache Set to true to retrieve the selection range from cache if editor doesn't own the focus now
+ * @returns A Range object of the selection range
+ */
+export const getSelectionRangeEx: GetSelectionRangeEx = (
+    core: EditorCore,
+    tryGetFromCache: boolean
```

Comment:
Maybe we can totally remove this parameter. I checked the code and there are only 3 places passing false to this param.

For those cases we can check one by one on how to do without this parameter